MRETestBed/Assets/TestBed Assets/Shaders/MREDiffuseVertex.shader

```diff
@@ -8,19 +8,17 @@ Shader "MRE/DiffuseVertex" {
 		_EmissiveTex ("Emissive Texture", 2D) = "white" {}
 
 		// Blend settings
-		_AlphaCutoff("Alpha Cutoff", Range(0,1)) = 0.5
-		[HideInInspector] _ShouldCutout("__should_cutout", Float) = 0.0
 		[HideInInspector] _SrcBlend("__src", Float) = 1.0
 		[HideInInspector] _DstBlend("__dst", Float) = 0.0
 		[HideInInspector] _ZWrite("__zw", Float) = 1.0
 	}
 	SubShader {
-		Tags { "RenderMode" = "Opaque" "Queue" = "Geometry" }
+		Tags { "RenderMode" = "Transparent" "Queue" = "Geometry" }
```

Comment:
Overdraw is the death of performance on Android especially. I don't think we can enable this by default. Also, if we did want this, we should change this in the SDK and not in the shader (`material.alphaMode = MRE.AlphaMode.Blend`).

Reply:
Oh are you saying that alpha blending is already supported in the SDK? If so, these changes are completely unnecessary. Alpha blending is the high order bit here. Mipmaps are nice to have.

Reply:
Yes, it's already supported. The RenderMode and everything else gets set in the MaterialPatcher.